build-system/compile/log-messages.js

```diff
@@ -33,23 +33,33 @@ const formats = {
   [`${pathPrefix}.simple.json`]: ({message}) => message,
 };
 
-/** @return {!Promise<!Array<!Object>>} */
-const extractedItems = () => fs.readJson(extractedPath).then(Object.values);
+/** @return {?Promise<!Array<!Object>>} */
```

Comment:
don't async functions technically always return a Promise? so it would actually be `{!Promise<!Array<!Object>|null>}`

Reply:
Nice catch.